gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -453,180 +500,209 @@ class _VcfSource(_TextSource):
 
   def __init__(self,
                file_pattern,
-               min_bundle_size=0,
                compression_type=CompressionTypes.AUTO,
                buffer_size=DEFAULT_VCF_READ_BUFFER_SIZE,
                validate=True):
-    super(_VcfSource, self).__init__(
-        file_pattern,
-        min_bundle_size,
-        compression_type,
-        strip_trailing_newlines=True,
-        coder=coders.StrUtf8Coder(),  # VCF files are UTF-8 per schema.
-        buffer_size=buffer_size,
-        validate=validate,
-        skip_header_lines=0,
-        header_matcher_predicate=lambda x: x.startswith('#'))
-    self._header_lines_per_file = {}
+    super(_VcfSource, self).__init__(file_pattern,
+                                     compression_type=compression_type,
+                                     validate=validate)
 
-  def process_header_lines_matching_predicate(self, file_name, header_lines):
-    self._header_lines_per_file[file_name] = header_lines
+    self._header_lines_per_file = {}
```

Comment:
Maybe remove this unused line from here?

Reply:
Done.